inc/iso22133object.hpp

```diff
@@ -80,8 +80,22 @@ class TestObject {
 	uint32_t getServerID() const { return serverID; }
 	ObjectSettingsType getObjectSettings() const { return objectSettings; }
 
-	int handleUDPMessage(char *buffer, int bufferLen, int udpSocket, char* addr, const uint32_t port);
-	int handleTCPMessage(char *buffer, int bufferLen);
+
+	/** SWIG Wrappers **/
+	bool isAwaitingFirstHeab() { return awaitingFirstHeab;}
+	//! Set the endpoint for the process channel
+	void setProcessChannelEndpoint(int udpSocket, char *addr, const uint32_t port);
+	//! Wrapper for handling function that converts to char vector
+	int handleMessage(char *buffer, int bufferLen);
```

Comment:
You have two functions declared with the same name, "handleMessage". Here and on row 181.

Reply:
I'd assume it's overridden?

Reply:
Yeah it's defined correctly in iso22133object.cpp